DEVELOPERS.md

```
@@ -358,3 +358,279 @@ Short list of documentation improvements (edit here as items land):
+
+**Do this:**
+```js
+module.exports = require('@fabric/http/functions/httpSharedMode');
+```
+
+**Do not do this:** copy the file into a second repo, or add a new `types/Foo.js` that collides with core `Foo`.
+
+---
+
+### Name collisions
+| Name | Canonical | Other copy | Relation | Action |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)